### DISCUSSION TITLE
I am not able to do quick add : wordpress anymore after phpAdmin update

### DISCUSSION THREAD

**Discussion by Ringoinmel** [leokhoa/laragon#913] [Q&A]
<img width="271" alt="Screenshot 2024-11-20 211719" src="https://github.com/user-attachments/assets/8c9be93e-6f8b-42f8-bf58-32d557041c26">

This is the error.  I got no problem before because there is no password to the root.  Any idea to fix it?  Thanks

**Comment by MTiberiu** [comment]
I had the same problem, the fix for me was to close the discord app

**Reply by Ringoinmel** [reply]
I haven't got discord app in my PC
